Apps/Playground/node_modules/react-native-babylon/android/src/main/cpp/BabylonNativeInterop.cpp

```diff
@@ -64,6 +68,8 @@ namespace Babylon
         ~Native()
         {
             Plugins::NativeEngine::DeinitializeGraphics();
+            // TODO: Figure out why this causes the app to crash
+            //Napi::Detach(m_env);
```

Comment:
Does this still happen?

Reply:
Yes, this is actually why when I tried to integrate your JSC changes it was crashing in some bgfx cleanup ('cause I also had this line). I just don't want to dig into it now given the JSI work.